# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpart/errno.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/disku.c.org	Thu May 15 18:31:53 2003
+++ ./src/disku.c	Thu May 15 18:32:02 2003
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/ioctl.h>
+#include <errno.h>
 #include "gpart.h"
 
 #if defined(__linux__)
@@ -27,7 +28,6 @@
 #endif
 
 #if defined(__FreeBSD__)
-#include <errno.h>
 #include <sys/disklabel.h>
 #endif
 
--- ./src/l64seek.c.orig	2003-05-20 11:33:12.000000000 +0200
+++ ./src/l64seek.c	2003-05-20 11:33:33.000000000 +0200
@@ -15,6 +15,7 @@
  */
 
 #include "l64seek.h"
+#include <errno.h>
 
 
 #define OSTACKLEN	16
